store.go

```diff
@@ -235,6 +236,73 @@ func (store *Store) Delete(ctx context.Context, prefix string, index int64) (ok
 	return
 }
 
+// Session return the session used to lock the given key. It returns an error
+// if the key is not locked or does not exist.
+func (store *Store) Session(ctx context.Context, key string) (sess Session, err error) {
+	var keyData KeyData
+	if keyData, err = store.readKeyData(ctx, key); err != nil {
+		return
+	}
+
+	if len(keyData.Session) == 0 {
+		err = fmt.Errorf("key %s is not locked", key)
+		return
+	}
+
+	var res io.ReadCloser
+	var configs []sessionConfig
+
+	path := "/v1/session/info/" + string(keyData.Session)
```

Comment:
How about moving this with the other variables just above and declare it with `var path = "/v1/session/info/" + string(keyData.Session)` so we keep the local style more consistent?